src/protocol_pro_2.cpp

```diff
@@ -235,4 +283,60 @@ void Pro2ProtocolObject::motors_control_loop(int sleeptime) {
   }
 }
 
+void Pro2ProtocolObject::update_params(std::string replacing_key,
```

Comment:
Probably should put all software related to editing, accessing, and modifying the params file in its own utilities.cpp file or something similar.